### ISSUE TITLE
Clippy lints for rust 1.53 fails on procmacros

### ISSUE THREAD

**Issue by simlay** [infinyon/node-bindgen#166]
Over in [fluvio-client-node](https://github.com/infinyon/fluvio-client-node/), in CI we get:
```
error: comparing with null is better expressed by the `.is_null()` method
   --> src/consumer.rs:101:5
    |
101 |     #[node_bindgen(mt)]
    |     ^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D clippy::cmp-null` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#cmp_null
    = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: comparing with null is better expressed by the `.is_null()` method
   --> src/consumer.rs:101:5
    |
101 |     #[node_bindgen(mt)]
    |     ^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D clippy::cmp-null` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#cmp_null
    = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error
```

**Comment by sehz** [comment]
@nicholastmosher can you look at this?

**Comment by simlay** [comment]
From what I can tell, this generation comes from the [function generation in `cb_complete`](https://github.com/infinyon/node-bindgen/blob/572d56abd5a716625aea4e341f12557ab9049b0b/nj-derive/src/generator/function.rs#L390) but it's unclear.

I'm also not sure how to run clippy on the trybuild ui-tests. Thoughts?

**Comment by sehz** [comment]
Maybe add clippy on Test examples

**Comment by simlay** [comment]
Closed in #165
